From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: vigaz <pudimdebacon3@gmail.com>
Date: Thu, 1 Oct 2026 22:10:19 -0300
Subject: [PATCH 2/2] Axion: include BoardConfigAxion.mk

Needs device/axion/common in the manifest. This was the dead include
f49b204 dropped from the base (broke every build that didn't have that
repo); belongs only in the Axion-specific build.
---
 BoardConfig.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/BoardConfig.mk b/BoardConfig.mk
index a9ca6da..5eb6cd6 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -24,6 +24,9 @@ include device/motorola/mt6768-common/BoardConfigCommon.mk
 
 TARGET_BOARD_PLATFORM := mt6768
 
+# Axion BoardConfig
+include device/axion/common/build/BoardConfigAxion.mk
+
 # Sepolicy
 SYSTEM_EXT_PRIVATE_SEPOLICY_DIRS += $(DEVICE_PATH)/sepolicy/private
 
